# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1502044014 25200
#      Sun Aug 06 11:26:54 2017 -0700
# Node ID 9f31de57ad36699b3f15f81367dedcc2ca9f1f60
# Parent  ca4cca5710dcbb18849c76e67b36d789161d9479
Include recently added test files in the source distribution

* test/module.mk (TEST_FILES): Include %reldir%/deprecate-props.tst in the list.
(%reldir%/bug-51192/module.mk): Include makefile snippet.

diff --git a/test/module.mk b/test/module.mk
--- a/test/module.mk
+++ b/test/module.mk
@@ -10,16 +10,17 @@ TEST_FILES += \
   %reldir%/bug-31371.tst \
   %reldir%/bug-38565.tst \
   %reldir%/bug-38576.tst \
   %reldir%/bug-46330.tst \
   %reldir%/bug-49904.tst \
   %reldir%/colormaps.tst \
   %reldir%/command.tst \
   %reldir%/complex.tst \
+  %reldir%/deprecate-props.tst \
   %reldir%/diag-perm.tst \
   %reldir%/error.tst \
   %reldir%/eval-catch.tst \
   %reldir%/for.tst \
   %reldir%/func.tst \
   %reldir%/global.tst \
   %reldir%/if.tst \
   %reldir%/index.tst \
@@ -50,20 +51,21 @@ include %reldir%/bug-36025/module.mk
 include %reldir%/bug-38236/module.mk
 include %reldir%/bug-38691/module.mk
 include %reldir%/bug-41723/module.mk
 include %reldir%/bug-44940/module.mk
 include %reldir%/bug-46660/module.mk
 include %reldir%/bug-50014/module.mk
 include %reldir%/bug-50035/module.mk
 include %reldir%/bug-50716/module.mk
+include %reldir%/bug-51192/module.mk
 include %reldir%/bug-51599/module.mk
 include %reldir%/class-concat/module.mk
+include %reldir%/classdef-multiple-inheritance/module.mk
 include %reldir%/classdef/module.mk
-include %reldir%/classdef-multiple-inheritance/module.mk
 include %reldir%/classes/module.mk
 include %reldir%/ctor-vs-method/module.mk
 include %reldir%/fcn-handle-derived-resolution/module.mk
 include %reldir%/nest/module.mk
 include %reldir%/publish/module.mk
 
 ALL_LOCAL_TARGETS += %reldir%/.gdbinit
 
